src/Components/SavedResponses.js

```diff
@@ -0,0 +1,95 @@
+// Modules
+import firebase from '../firebase';
+import { useEffect, useState } from 'react';
+import { getDatabase, ref, onValue } from 'firebase/database';
+import { FaRegHeart, FaRegSmile, FaRegFrown, FaRegMeh, FaArrowUp } from 'react-icons/fa';
+
+const PastResponses = () => {
+  // Defining State
+  const [userData, setUserData] = useState([]);
+  const [count, setCounter] = useState(6);
+  // Firebase Connection
+  useEffect(() => {
+    // Variable to hold database details:
+    const database = getDatabase(firebase);
+    // Variable that references our database:
+    const dbRef = ref(database);
+    // Event listener that will fire from the database & call the data 'response'
+    onValue(dbRef, (response) => {
+      // Variable to store the new state we're creating in our app
+      const newState = [];
+      // Store our response,val() in a variable;
+      const dataResponse = response.val();
+      // Data is an object so we iterate using a for-in loop to access each tarot card object.
+      for (let key in dataResponse) {
+        // Push each tarot card object to an array we already created in newState
+        newState.push({ key: key, tarotCard: dataResponse[key] });
+      }
+      // Call setUserData to update our components state using the local array newState:
+      setUserData(newState);
+    })
+  }, [])
+
+  // Show More Results Button
```

Comment:
This is v clever! Maybe a stretch goal after bootcamp would be to implement infinite scroll since that's a big draw of react!

Reply:
Love this idea! I'll add it to my list.